src/utilities/view.ts

```diff
@@ -17,13 +19,30 @@ import {
 /**
  * Generates a default generic UI view.
  */
-export function generateDefaultView(activityTypes: ActivityType[] = [], resourceTypes: ResourceType[] = []): View {
+export function generateDefaultView(
+  activityTypes: ActivityType[] = [],
+  resourceTypes: ResourceType[] = [],
+  externalEventTypes: ExternalEventType[] = [],
+): View {
   const now = new Date().toISOString();
   const types: string[] = activityTypes.map(({ name }) => name);
 
   const timeline = createTimeline([], { marginLeft: 250, marginRight: 30 });
   const timelines = [timeline];
 
+  const externalEventLayer = createTimelineExternalEventLayer(timelines, {
```

Comment:
Consider not adding a row dedicated to external events if no external events are found in this instance of aerie? Alternatively, consider moving the external events row below the default `Activities by Type` row since at least for the time being, existing aerie users are not set up to make use of external events.

Reply:
This is addressed in [this commit](https://github.com/NASA-AMMOS/aerie-ui/pull/1396/commits/79898b9b3ba677083748bb26fa977a01b4795430). We show the row only if derivation groups with external events in them are associated with the plan. If a derivation group is empty, the row is not created. We can modify this in case it is preferred that this row shows even if a derivation group is empty (i.e. it was created but no sources were associated); this may be useful to speed up the process for users setting up views prior to uploading many, large files.